### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by iamlordvodemort**
@Nigh7Sh4de can you take a look at this for me. I am not sure the best way to get the profile info to the web page from the service class.
I also made some minor changes to separate the "tabbarcontroller" from the index.html file.

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -18,15 +19,49 @@
...
-
-
     <link href="lib/ionic/css/ionic.css" rel="stylesheet">
     <link href="lib/ion-datetime-picker/ion-datetime-picker.min.css" rel="stylesheet">
     <link href="css/style.css" rel="stylesheet">
 
     <!-- IF using Sass (run gulp sass first), then uncomment below and remove the CSS includes above
     <link href="css/ionic.app.css" rel="stylesheet">
     -->
+    <style type="text/css">
```
Comment (Nigh7Sh4de) [inline www/index.html]: Please move this out into an external css file

File: www/index.html [hunk 2]
```diff
@@ -49,9 +84,24 @@
     <script src="js/homeCtrl.js"></script>
     <script src="js/settingsCtrl.js"></script>
     <script src="js/resourceService.js"></script>
-  </head>
-  <body ng-app="starter">
+    <script src="js/userInfoService.js"></script>
+</head>
+
+<body ng-app="starter" animation="slide-left-right-ios7">
+    <div>
```
Comment (Nigh7Sh4de) [inline www/index.html]: Any reason for these nested div's?

File: www/index.html [hunk 3]
```diff
@@ -60,13 +110,9 @@
         <a class="tab-item" href="#/reservations">
             Reservations
         </a>
-        <a class="tab-item" href="#/manage">
```
Comment (Nigh7Sh4de) [inline www/index.html]: Why did you get rid of the manage tab?

File: www/js/app.js [hunk 4]
```diff
@@ -29,34 +30,58 @@ var app = angular.module('starter',
...
             .state('resourceList', {
                 url: '/manage',
                 templateUrl: 'templates/resources.html',
                 controller: 'ResourceList'
             })
-            .state('settings', {
+
+            /** Tab bar routes **/
+            .state('tabsController.home', {
+                url: '/homePage',
```
Comment (Nigh7Sh4de) [inline www/js/app.js]: Convention is to have the landing page either on route `/` or `/home` or `/index`. I personally prefer `/`, however feel free to update to fit any of these conventions.

File: www/js/settingsCtrl.js [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+        $scope.modal = modal;
+    })
+
+	$scope.manage = function() {
+    	$scope.modal.show();
+  	}
+
+  	$scope.$on('$ionicView.enter', function() {
+     // Code you want executed every time view is opened
+     console.log('Opened!')
```
Comment (Nigh7Sh4de) [inline www/js/settingsCtrl.js]: Do not commit console logs please! If you are commiting an unfinished feature with an unfinished function, please use `console.error('Not Implemented', ...)`

File: www/js/userInfoService.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
+var OFFLINE_ONLY = true;
```
Comment (Nigh7Sh4de) [inline www/js/userInfoService.js:1]: This is fine for now. But we should create a .gitignore'd config file that specifies whether to run in demo or production mode.

File: www/templates/tabsController.html [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<ion-view id="tabsBarController">
+  <ion-tabs class="tabs-stable tabs-icon-top" id="tabsController-tabs1">
+    <ion-tab title="Parking" icon="ion-android-car" href="#/tabsBarController/home" id="tabsController-tab1">
```
Comment (Nigh7Sh4de) [inline www/templates/tabsController.html]: Please have a more descriptive `id` for each `ion-tab`

**Review by Nigh7Sh4de — DISMISSED** [review]
I left a few comments for things I'd like you to change/reconsider. Let me know if you disagree with anything.

File: www/index.html (hunk 1 above)
Comment (iamlordvodemort) [reply www/index.html]: Done!

File: www/index.html (hunk 2 above)
Comment (iamlordvodemort) [reply www/index.html]: I got it from a template when i created the UI in ionic creator. I've removed them

File: www/js/userInfoService.js (hunk 6 above)
Comment (iamlordvodemort) [reply www/js/userInfoService.js:1]: Yah, I'm just following your examples on this... but a demo/production configuration would definitely work for the future

File: www/index.html (hunk 3 above)
Comment (iamlordvodemort) [reply www/index.html]: This is just temporary, for some reason I kept seeing this page overriding the tab bar. I'll fix it before I created a PR to be merged.

**Comment by iamlordvodemort** [comment]
@Nigh7Sh4de can you give some guidance as to my original question of making the service request and updating the UI on the settings page with the profile name ?

**Comment by Nigh7Sh4de** [comment]
Oh sorry I did not notice your question. Yeah so in the settings controller when you resolve the promise to get the user info, just populate some `$scope` variable with the result. Something like:
```
userInfoService.getProfileInfo()
.then(function(userInfo) {
    $scope.userInfo = userInfo;
})
.catch(function(err) { /*some sort of error handling*/ });
```

**Comment by iamlordvodemort** [comment]
@Nigh7Sh4de updated PR, should be good to go now..

**Pull request merged**